projects/client/RabbitMQ.Client/src/client/impl/AutorecoveringConnection.cs

```diff
@@ -961,8 +960,32 @@ private enum RecoveryConnectionState
         private Task _recoveryTask;
         private RecoveryConnectionState _recoveryLoopState = RecoveryConnectionState.Connected;
 
-        private readonly ConcurrentQueue<RecoveryCommand> _recoveryLoopCommandQueue = new ConcurrentQueue<RecoveryCommand>();
-        private readonly SemaphoreSlim _semaphore = new SemaphoreSlim(0);
+
+        private class AsyncConcurrentQueue<T>
+        {
+            private readonly ConcurrentQueue<T> _internalQueue = new ConcurrentQueue<T>();
+            private readonly SemaphoreSlim _semaphore = new SemaphoreSlim(0);
+
+            public async Task<T> DequeueAsync(CancellationToken token)
```

Comment:
Use a default value for this parameter and set it to `default` (standard practice).

Reply:
Done